src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,144 @@
+import { useEffect, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import {
+  getFatherPerson,
+  getMotherPerson,
+  hasIncludeQuery,
+} from '../utils/utils';
+import { PeopleFilters } from '../components/PeopleFilters';
+
+export const PeoplePage = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isErrorHappened, setIsErrorHappened] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+
+  const { search } = useLocation();
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then((peopleFromServer) => {
+        setPeople(
+          peopleFromServer.map((personData) => {
+            const mother = getMotherPerson(
+              peopleFromServer, personData,
+            );
+            const father = getFatherPerson(
+              peopleFromServer, personData,
+            );
+
+            const person: Person = {
+              ...personData,
+              mother,
+              father,
+            };
+
+            return person;
+          }),
+        );
+      })
+      .catch(() => setIsErrorHappened(true))
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  const getFilteredPeople = () => {
+    const params = new URLSearchParams(search);
+
+    const sex = params.get('sex') || '';
+    const centuries = params.getAll('centuries') || [];
+    const query = params.get('query') || '';
+    const sort = params.get('sort') || '';
+    const order = params.get('order') || '';
+
+    let preparedPeople = [...people];
+
+    if (sex.length > 0) {
```

Comment:
```suggestion
    if (!!sex.length) {
```

Reply:
I wanted to do this but my linter said  `'Redundant double negation'`